<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DemoMapBounds</class>
 <widget class="QWidget" name="TestMapBounds">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0">
       <item>
        <widget class="PwGisWidget" name="pwgiswidget" native="true">
         <property name="minimumSize">
          <size>
           <width>600</width>
           <height>340</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="sizeConstraint">
          <enum>QLayout::SetFixedSize</enum>
         </property>
         <item row="0" column="0" rowspan="4">
          <widget class="QPlainTextEdit" name="plainTextEdit"/>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="btnGetMapBounds">
           <property name="text">
            <string>get Map Bounds</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="btnGetViewBounds">
           <property name="text">
            <string>get View Bounds</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="btnGetZoom">
           <property name="text">
            <string>get Current Zoom</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QPushButton" name="btnGetZoomInfo">
           <property name="text">
            <string>get Zoom Info</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <property name="sizeConstraint">
          <enum>QLayout::SetFixedSize</enum>
         </property>
         <item row="1" column="0" colspan="2">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Bottom:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLineEdit" name="editLeft">
           <property name="toolTip">
            <string/>
           </property>
           <property name="statusTip">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Left:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLineEdit" name="editBottom">
           <property name="toolTip">
            <string/>
           </property>
          </widget>
         </item>
         <item row="16" column="2">
          <widget class="QLineEdit" name="editSetZoom"/>
         </item>
         <item row="17" column="0" colspan="3">
          <widget class="QPushButton" name="btnSetZoom">
           <property name="text">
            <string>set Current Zoom</string>
           </property>
          </widget>
         </item>
         <item row="14" column="0" colspan="3">
          <widget class="QPushButton" name="btnSetMapCenter">
           <property name="text">
            <string>setMapCenter</string>
           </property>
          </widget>
         </item>
         <item row="13" column="1" colspan="2">
          <widget class="QLineEdit" name="editCenterLat"/>
         </item>
         <item row="12" column="1" colspan="2">
          <widget class="QLineEdit" name="editCenterLon"/>
         </item>
         <item row="13" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Lat:</string>
           </property>
          </widget>
         </item>
         <item row="10" column="0" colspan="3">
          <widget class="QPushButton" name="btnZoomMapToByOffset">
           <property name="text">
            <string>ZoomMapTo By Offset</string>
           </property>
          </widget>
         </item>
         <item row="12" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Lon:</string>
           </property>
          </widget>
         </item>
         <item row="9" column="2">
          <widget class="QLineEdit" name="editOffsetR"/>
         </item>
         <item row="9" column="0" colspan="2">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>OffsetR:</string>
           </property>
          </widget>
         </item>
         <item row="8" column="2">
          <widget class="QLineEdit" name="editOffsetB"/>
         </item>
         <item row="8" column="0" colspan="2">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>OffsetB:</string>
           </property>
          </widget>
         </item>
         <item row="7" column="2">
          <widget class="QLineEdit" name="editLat"/>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Lat:</string>
           </property>
          </widget>
         </item>
         <item row="6" column="2">
          <widget class="QLineEdit" name="editLon"/>
         </item>
         <item row="4" column="0" colspan="3">
          <widget class="QPushButton" name="btnZoomMapTo">
           <property name="text">
            <string>ZoomMapTo</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Lon:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLineEdit" name="editTop">
           <property name="toolTip">
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLineEdit" name="editRight">
           <property name="toolTip">
            <string/>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Top:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Right:</string>
           </property>
          </widget>
         </item>
         <item row="11" column="0">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="15" column="0">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="5" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="18" column="0">
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <property name="sizeConstraint">
          <enum>QLayout::SetFixedSize</enum>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Current zoom:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="editZoom">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PwGisWidget</class>
   <extends>QWidget</extends>
   <header location="global">PwGis/pwgiswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
